var/spack/repos/builtin/packages/form/package.py

```diff
@@ -10,18 +10,18 @@ class Form(AutotoolsPackage):
     """FORM is a Symbolic Manipulation System."""
 
     homepage = "https://www.nikhef.nl/~form/"
-    url      = "https://github.com/vermaseren/form/archive/v4.2.1.tar.gz"
+    url      = "https://github.com/vermaseren/form/releases/download/v4.2.1/form-4.2.1.tar.gz"
     maintainers = ['iarspider']
 
-    version('4.2.1', sha256='6f32c7470d00e8ab6934dc352f5a78e29290146a00e5775f8cd5fef7810bbbb8')
-    version('4.1-20131025', sha256='caece2c6e605ccf32eb3612c4ed5c9257a7a62824ad219c5e46b6d00066f1ba6')
+    version('4.2.1', sha256='f2722d6d4ccb034e01cf786d55342e1c21ff55b182a4825adf05d50702ab1a28')
+    version('4.1-20131025', sha256='fb3470937d66ed5cb1af896b15058836d2c805d767adac1b9073ed2df731cbe9')
```

Comment:
```suggestion
    version('4.1-20131025', sha256='fb3470937d66ed5cb1af896b15058836d2c805d767adac1b9073ed2df731cbe9',
            url='https://github.com/vermaseren/form/releases/download/v4.1-20131025/form-4.1.tar.gz')
```

and remove `url_for_version` function

Reply:
Thank you for your review. I will make a change following your suggestion.

Reply:
You don't need `url_for_version` function if you pass `url` parameter

Reply:
AFAICS, this is pushed now as commmit 67165dd by Takahiro

Reply:
Yup. Still getting used to reviewing, sorry!